newIDE/app/public/external/utils/path-editor.js

```diff
@@ -124,15 +127,17 @@ const selectBaseFolderPath = headerObject => {
   if (!selectedDir) {
     return;
   }
-  if (!selectedDir.toString().startsWith(state.projectBasePath)) {
+  const selectedDirClean = selectedDir[0].replace(/\\/gi,'/')
```

Comment:
Could you try to see if you can use `path` functions instead (from `require('path')`) (https://nodejs.org/api/path.html).
This kind of manipulation of paths manually is in general fragile against edge cases.
I think you can use `path.normalize` (or `path.posix.normalize`)

Reply:
ah good idea :)